backend/app/controllers/controller.js

```diff
@@ -0,0 +1,55 @@
+/* This is the controller file. Its functions are called when 
+a route is navigated to. These functions in turn call upon the 
+database accessor to get the requested information out of the 
+database to return to the front end. */
```

Comment:
Bridge of comunication between Route and DB accessor.

Route <--> Controller <-->  DB Accessor